habitat-lab/habitat/tasks/rearrange/multi_task/task_creator_utils.py

```diff
@@ -43,21 +45,25 @@ def create_task_object(
     task_cls = registry.get_task(task_cls_name)
 
     config = copy.deepcopy(cur_config)
-    with habitat.config.read_write(config):
-        if task_config_path is not None:
-            pass_args: List[Any] = []
-            for k, v in task_config_args.items():
-                pass_args.extend((k, v))
-            task_config = habitat.get_config(
-                osp.join(TASK_CONFIGS_DIR, task_config_path + ".yaml"),
-                pass_args,
-            )
+
+    if task_config_path is not None:
+        pass_args: List[Any] = []
+        for k, v in task_config_args.items():
+            pass_args.append(f"{k}={v}")
```

Comment:
```suggestion
        pass_args: List[str] = [f"{k}={v}" for k, v in task_config_args.items()]
```
faster, more efficient, also I think this makes the explicit specifying of the List type unnecessary.